package.json

```diff
@@ -27,9 +27,9 @@
     "test:serverside": "yarn build && node src/serverSideTest.js",
     "storybook": "start-storybook -p 9009",
     "storybook:deploy": "storybook-to-ghpages",
-    "build": "tsc && webpack --progress",
+    "build": "tsc -p tsconfig.build.json && webpack --progress",
     "build:watch": "tsc && webpack --watch",
-    "lint": "tsc --noEmit --emitDeclarationOnly false && eslint --ext js,jsx,ts,tsx src && stylelint \"src/**/*.{css,scss}\"",
```

Comment:
return to how this used to be, since the base tsconfig that is used by default has `noEmit: true` already set